<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabManagerWidget</class>
 <widget class="QWidget" name="TabManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tab Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelGroup">
       <property name="text">
        <string>Group By:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groupTypeComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MacToolButton" name="sideBySide">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="tabmanagerplugin.qrc">
         <normaloff>:/tabmanager/data/side-by-side.png</normaloff>:/tabmanager/data/side-by-side.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="MacToolButton" name="bookmarkSelection">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="tabmanagerplugin.qrc">
         <normaloff>:/tabmanager/data/tab-bookmark.png</normaloff>:/tabmanager/data/tab-bookmark.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="MacToolButton" name="closeSelection">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="tabmanagerplugin.qrc">
         <normaloff>:/tabmanager/data/tab-close.png</normaloff>:/tabmanager/data/tab-close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="MacToolButton" name="detachSelection">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="tabmanagerplugin.qrc">
         <normaloff>:/tabmanager/data/tab-detach.png</normaloff>:/tabmanager/data/tab-detach.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MacToolButton</class>
   <extends>QToolButton</extends>
   <header>mactoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>groupTypeComboBox</tabstop>
  <tabstop>closeSelection</tabstop>
  <tabstop>detachSelection</tabstop>
  <tabstop>bookmarkSelection</tabstop>
 </tabstops>
 <resources>
  <include location="tabmanagerplugin.qrc"/>
 </resources>
 <connections/>
</ui>
